domains/facileachercher.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "rastaff635@gmail.com"
```

username is required
```suggestion
    "username": "rst0203",
    "email": "rastaff635@gmail.com"
```